README.md

```diff
@@ -1,2 +1,78 @@
+It also contains other associated charm libraries and utilities that make up the Ceph ecosystem that
+can be deployed via Juju.
+
+These charms used to reside in their own repository in the past. The reason they have been consolidated
+into a single repository is to make larger changes that involve multiple components easier.
+
+There are nine individual charms in this monorepo, each defining and deploying a specific Ceph application.
+However, these charms often work together, i.e. they are often deployed together via Juju integrations.
+For example, the ceph-osd and ceph-mon charms are often deployed together to provide and scale storage in a Ceph cluster.
+
+# The charm index
+
+1. [ceph dashboard](https://github.com/canonical/ceph-charms/tree/main/ceph-dashboard)
+2. [ceph-fs](https://github.com/canonical/ceph-charms/tree/main/ceph-fs)
+3. [ceph-mon](https://github.com/canonical/ceph-charms/tree/main/ceph-mon)
+3. [ceph-nfs](https://github.com/canonical/ceph-charms/tree/main/ceph-nfs)
+5. [ceph-nvme](https://github.com/canonical/ceph-charms/tree/main/ceph-nvme)
+6. [ceph-osd](https://github.com/canonical/ceph-charms/tree/main/ceph-osd)
+7. [ceph-proxy](https://github.com/canonical/ceph-charms/tree/main/ceph-proxy)
+8. [ceph-radosgw](https://github.com/canonical/ceph-charms/tree/main/ceph-radosgw)
+9. [ceph-rbd-mirror](https://github.com/canonical/ceph-charms/tree/main/ceph-rbd-mirror)
+
+# Additional sub-projects
+
+In addition to charms, the repository contains the following:
+
+- [charms.ceph](https://github.com/canonical/ceph-charms/tree/main/charms.ceph): Common support library used by the charms.
+- [constraints](https://github.com/canonical/ceph-charms/tree/main/constraints): List of common dependencies shared by the charms.
+- [terraform](https://github.com/canonical/ceph-charms/tree/main/terraform): Maintains terraform modules used for charm deployments.
+- [tests](https://github.com/canonical/ceph-charms/tree/main/tests): Miscellaneous scripts used to facilitate testing.
+
+# Documentation
+
+Each individual charm contains a README section with details about the specific charm, including its description, functionality, usage and contributing guide.
+
+See [charmhub](https://charmhub.io/?q=ceph) for more information about the charms.
+
+The [OpenStack charm guide](https://docs.openstack.org/charm-guide)provides guides for working with charms, including development and support notes.
+
+The [OpenStack charms deployment guide](https://docs.openstack.org/project-deploy-guide/charm-deployment-guide) demonstrates how to deploy and configure charms manually to build an OpenStack cloud.
+
+# Project and community
+
+We warmly welcome community contributions, suggestions, fixes, and constructive feedback.
+If you find any errors or have suggestions for improvements, please report a bug on launchpad on the specific component (For example, for ceph dashboard, use [this link](https://bugs.launchpad.net/charm-ceph-dashboard).
```

Comment:
[* suggested rewording *] "... please [report a bug on Launchpad](https://bugs.launchpad.net/) for the specific component; for example, [report a `ceph-dashboard-charm` bug](https://bugs.launchpad.net/charm-ceph-dashboard) in the Ceph Dashboard charm project on Launchpad.